drydock_backups/plugin.py

```diff
@@ -99,3 +99,10 @@
     encoding="utf-8",
 ) as fi:
 
+
```

Comment:
The 2 empty lines [here](https://github.com/eduNEXT/drydock-backups/blob/hpg/redwood-upgrade/drydock_backups/plugin.py#L101-L102) are strange, could we remove them?